.github/workflows/build.yaml

```
@@ -0,0 +1,190 @@
+      docker_image: ${{ steps.save_docker_image.outputs.docker_image }}
+      db_image: ${{ steps.save_db_image.outputs.db_image }}
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Cache Docker Layers
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)